src/triggers/issue.ts

```diff
@@ -80,7 +80,7 @@ const buildIssueList = (orderBy: "createdAt" | "updatedAt") => async (z: ZObject
         statusId: bundle.inputData.status_id,
         creatorId: bundle.inputData.creator_id,
         assigneeId: bundle.inputData.assignee_id,
-        priority: bundle.inputData.priority,
+        priority: parseFloat(bundle.inputData.priority),
```

Comment:
```suggestion
        priority: parseInt(bundle.inputData.priority),
```

Reply:
I'm down to accept this. The reason I opted for `parseFloat` was that the [graphql schema](https://studio.apollographql.com/public/Linear-API/schema/reference/objects/Issue?variant=current#priority) seems to describe issue priority as a Float and I didn't want it to break should Linear actually decide to start using non-integer numbers for priority at some point in the future. Do you have any inside information on that?

Reply:
> Do you have any inside information on that?

Yep, working there 😄 

Thanks for pointing our our GraphQL schema is not in sync. We'll be fixing that (make field an integer).